docs/1_Getting_Started.md

```diff
@@ -32,7 +32,7 @@ See [Architecture Introduction]({{ site.baseurl }}{% link docs/2_Architecture.md
 ## Pre-requisites
 
 Make sure following tools are installed
-* NodeJS 8.X
+* NodeJS 8 (LTS), 9, or 10 (LTS) *we do not support higher versions as the dependancy cahin does not permit this*
```

Comment:
"cahin" would be chain I guess? :)

Reply:
😂 every time! ... fixed with a forced update ...